docs/en/development/build-osx.md

```diff
@@ -35,13 +35,22 @@ sudo xcode-select --install
 
 ## Install Required Compilers, Tools, and Libraries {#install-required-compilers-tools-and-libraries}
 
+:::note
```

Comment:
I am skeptical about such a note: it will need to be checked for validity and updated every time a new LLVM release is published. Do the issues with LLVM 19 (the latest version as of now) still persist? If yes, it would be better to fix them right away.

Reply:
few days ago i tried building with llvm 19 and it used to give me errors of some unused variable thing , on which i was advised to move to llvm 17 ,but not sure as of today whether latest version if fixed

Reply:
@rschu1ze see #27705

Reply:
Interesting. We should nevertheless only include "permanent" gotchas in the docs, for example the tip about case-sensitive vs. case-insensitive file systems. "Temporary" problems are better registered and described as [issues](https://github.com/ClickHouse/ClickHouse/issues).

Reply:
ok so should i remove the llvm note ??

Reply:
yes